plugins/modules/win_zip.ps1

```diff
@@ -51,6 +51,9 @@ catch {
 }
 
 Function Compress-Zip($src, $dest) {
+    # Disable using backslash for Zip path. This works for .NET 4.6.1 or later
+    [System.AppContext]::SetSwitch('Switch.System.IO.Compression.ZipFile.UseBackslash', $false)
```

Comment:
While pretty much most installs will have .NET 4.6.1 these days technically speaking we still support Server 2012 and 2012 R2 until they are end of life later this year. These versions ship with an older version of dotnet framework so we should have a check to see if `System.AppContext` exists before setting the switch.

```suggestion
    if ([object].Assembly.GetType("System.AppContext")) {
        [System.AppContext]::SetSwitch('Switch.System.IO.Compression.ZipFile.UseBackslash', $false)
    }
```

Reply:
@jborean93 Thank you for the review! Sure thing, i will add that to the PR